topics:
  - /camera/camera/color/image_raw/compressed

output_dir: ~/Workspace/Outputs/rosbags/test
storage: sqlite3
compression: none     #zstd
compression_mode: file

max_bag_size: 0
max_bag_duration: 0
qos_profile_overrides_path: ""
